subsys/bluetooth/audio/has.c

```diff
@@ -157,32 +156,35 @@ static struct bt_gatt_attr has_attrs[] = {
 };
 
 static struct bt_gatt_service has_svc;
-
-#if defined(CONFIG_BT_HAS_FEATURES_NOTIFIABLE)
-/* Features notification work */
-static void features_work_process(struct k_work *work);
-static K_WORK_DEFINE(features_work, features_work_process);
-
-#define FEATURES_ATTR &has_attrs[2]
-#if defined(CONFIG_BT_HAS_PRESET_SUPPORT)
-#define PRESET_CONTROL_POINT_ATTR &has_attrs[5]
-#define ACTIVE_PRESET_INDEX_ATTR &has_attrs[8]
-#endif /* CONFIG_BT_HAS_PRESET_SUPPORT */
-#else
-#if defined(CONFIG_BT_HAS_PRESET_SUPPORT)
-#define PRESET_CONTROL_POINT_ATTR &has_attrs[4]
-#define ACTIVE_PRESET_INDEX_ATTR &has_attrs[7]
-#endif /* CONFIG_BT_HAS_PRESET_SUPPORT */
-#endif /* CONFIG_BT_HAS_FEATURES_NOTIFIABLE */
+static struct bt_gatt_attr *hearing_aid_features_attr;
+static struct bt_gatt_attr *preset_control_point_attr;
+static struct bt_gatt_attr *active_preset_index_attr;
```

Comment:
Should these be stored in `static struct bt_has has;`?

Reply:
As `struct bt_has` is common for client and server I would not put it there as it would be unused by client.